mod/wet4/actions/object/question/autocomplete.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
The main reason I did it in our mod is to deal with features that we have(French/English content). At a later time we can go back and push this to the questions mod without our unique features in it.